pkg/pipelines/service.go

```diff
@@ -195,3 +200,19 @@ func createSvcImageBinding(cfg *config.PipelinesConfig, env *config.Environment,
 	resourceFilePath := makeImageBindingPath(cfg, filename)
 	return name, filename, res.Resources{resourceFilePath: triggers.CreateImageRepoBinding(cfg.Name, name, imageRepo, strconv.FormatBool(isTLSVerify))}
 }
+
+func createConfigFolder(m *config.Manifest, appFs afero.Fs, o *AddServiceOptions) error {
+	basePath, err := homedir.Expand(o.PipelinesFolderPath)
+	if err != nil {
+		return err
```

Comment:
how about expanding on this error a bit?

Getting the error [`cannot expand user-specific home dir`](https://github.com/mitchellh/go-homedir/blob/master/homedir.go#L68) probably isn't that useful to a user?